delivery_postlogistics/models/stock_picking.py

```diff
@@ -0,0 +1,242 @@
+# Copyright 2013-2016 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import base64
+from operator import attrgetter
+
+from odoo import _, api, exceptions, fields, models
+
+from ..postlogistics.web_service import PostlogisticsWebService
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    delivery_fixed_date = fields.Date(
+        "Fixed delivery date", help="Specific delivery date (ZAW3217)"
+    )
+    delivery_place = fields.Char(
```

Comment:
wouldn't be better to have new partner relation here?
But I see you preserve an original behavior. Not mandatory to change.

Reply:
> @phuctranerp what about the question by @jbaoudoux ?

@simahawk I dont know the correct endpoint for testing instance for this, and I cannot find it in postlogistics' document. However, afaik, environment now depends on the account type, as last time we use the production endpoint for testing. Any idea ?

Reply:
just leave it as this but add `TODO: consider refactoring these fields using a partner relation instead`

Reply:
updated